### PR TITLE
Copy more BetterLabel's attributes to the UIButton

### PR DESCRIPTION
**Pull request by LukasHromadnik**

### CODE REVIEW

**Comment by olejnjak** [comment]
What about some tests? 🤔

File: BetterLabelTests/UIButtonExtensionTests.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//
+
+import XCTest
+@testable import BetterLabel
+
+final class UIButtonExtensionTests: XCTestCase {
+    func testCallability() {
+        let label = BetterLabel()
+        let button = UIButton()
+        button.setBetterLabel(label, for: .normal)
```
Comment (olejnjak) [inline BetterLabelTests/UIButtonExtensionTests.swift:16]: To test that this code can be called, you should not import BetterLabel as `@testable`

File: BetterLabelTests/UIButtonExtensionTests.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    }
+    
+
```
Comment (olejnjak) [inline BetterLabelTests/UIButtonExtensionTests.swift]: I think that those spaces should be deleted.

**Review by olejnjak — CHANGES_REQUESTED** [review]

File: BetterLabelTests/UIButtonExtensionTests.swift (hunk 2 above)
Comment (LukasHromadnik) [reply BetterLabelTests/UIButtonExtensionTests.swift]: Yes, you're right. I've removed it

**Review by olejnjak — APPROVED** [review]

**Pull request merged**